### DISCUSSION TITLE
Drawn in pencil

### DISCUSSION THREAD

**Discussion by StepHaze** [lllyasviel/Fooocus#2289] [Q&A]
Is there a style as if a picture was drawn in colored pencil? Picture should be plain and inelaborate.

Examples: 
https://theoldkennels.co.uk/wp-content/uploads/2021/02/mushrooms-thumb-scaled-e1612477899131.jpg
https://i.etsystatic.com/18975340/r/il/10385a/2708240872/il_570xN.2708240872_coe4.jpg
https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEgqtgH1rPZz3eIQarmfjZbDhaclPmP8dCV7TJb-6ciDMC8DS0NFmoF2byiuD9zYqwaOrbEM_X7nZgCDNBYPZRLjcOxRZ0kGxoyfgsk-v0YtaX5Hvd2lOXtm9URuv73AMMnL7qpBd0KlDMH1x3LZf_xFIeHs5bZIOgCezzfAlhyEuK3xRTfVSTZiYdv6/s1750/15.jpg

**Comment by mashb1t** [comment]
Please check if style `Colored Pencil Art` fits your needs.
